walletunlocker/service_test.go

```diff
@@ -630,37 +648,9 @@ func doChangePassword(service *walletunlocker.UnlockerService, testDir string,
 	if !bytes.Equal(response.AdminMacaroon, testMac) {
 		errChan <- fmt.Errorf("mismatched macaroon: expected %x, got "+
 			"%x", testMac, response.AdminMacaroon)
-	}
-
-	// Close the macaroon DB and try to open it and read the root key with
-	// the new password.
-	store, err := openOrCreateTestMacStore(
```

Comment:
Why do we remove this validation? I think it makes sense to check that we actually have changed the password...

Reply:
But this re-opens the macaroon store with the new password, to make sure it was actually changed...

Reply:
Ah, wow... Yeah, you're right. Nice catch!